From 2297b79eafa7be24e71d339ccd476a65e1aaf6b5 Mon Sep 17 00:00:00 2001
From: renchenglei <chenglei.ren@intel.com>
Date: Wed, 23 Jan 2019 09:56:53 +0800
Subject: [PATCH] Fix typo error for vulkan library

The typo error will cause CTS dEQP vulkan module failure, this
commit help fix this issue.

Tracked-On: OAM-75410
Signed-off-by: Ren Chenglei <chenglei.ren@intel.com>
---
 sepolicy/graphics/project-celadon/file_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sepolicy/graphics/project-celadon/file_contexts b/sepolicy/graphics/project-celadon/file_contexts
index e5a49ec..ae94d63 100644
--- a/sepolicy/graphics/project-celadon/file_contexts
+++ b/sepolicy/graphics/project-celadon/file_contexts
@@ -13,7 +13,7 @@
 /(vendor|system/vendor)/lib(64)?/libdrm_intel_pri\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/hw/gralloc\.project-celadon\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/i965_dri\.so u:object_r:same_process_hal_file:s0
-/(vendor|system/vendor)/lib(64)?/hw/vulkan\.project-broxton\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/hw/vulkan\.project-celadon\.so u:object_r:same_process_hal_file:s0
 /vendor/bin/hw/android\.hardware\.graphics\.composer\.allocator@2\.1-service u:object_r:hal_graphics_composer_default_exec:s0
 /(vendor|system/vendor)/lib(64)?/libdrm_intel\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/libpciaccess\.so u:object_r:same_process_hal_file:s0
-- 
1.9.1

